lib/runner.js

```diff
@@ -75,12 +77,12 @@ module.exports = (locations = [], formatFn = require('pg-format'), isolationLeve
 
     const tableClause = tableSql(schema, table);
     const updateClause = sqlZip(',', update);
-    const whereClause = R.when(Boolean, R.concat('WHERE '), sqlZip(' AND ', where));
-
-    return _prepare(sqlFormatter, queries, '_update', [tableClause, updateClause, whereClause]);
+    const whereClause = R.when(Boolean, R.concat('WHERE '), sqlZip(' AND ', R.dissoc('_returning', options)));
+    const queryName = conditionalConcat('_update', '_returning', R.equals(options._returning, true));
```

Comment:
What about not using Ramda?
```suggestion
    const queryName = options._returning ? '_returning' : '_update';
```

Reply:
Good to me, but this should be:

```js
const queryName = options._returning ? '_update_returning' : '_update';
```

Reply:
You're right, I forgot about `conditionalConcat`.

I would like to start removing Ramda/lodash when possible.